Singular.js

```diff
@@ -114,4 +118,32 @@ export class Singular {
     static getLimitDataSharing() {
         return SingularBridge.getLimitDataSharing();
     }
+
+    // SKAN methods
+    static skanUpdateConversionValue(conversionValue) {
+        if (Platform.OS === 'ios') {
+            return SingularBridge.skanUpdateConversionValue(conversionValue);
+        }
+
+        console.log(PLATFORM_NOT_SUPPORTED)
```

Comment:
I'm not sure about adding the log...
Because it can confuse clients with react-native logs.
Usually in SDKs logs are opt-in.
I think it's safe to assume that they will know these method won't work in Android so we can avoid logging.
Wdyt?